.cc-metadata.yml

```diff
@@ -1,7 +1,7 @@
 # Whether this GitHub repo is engineering related
 engineering_project: true
 # Name of the repository/project in English
-english_name: CC Search (Legacy)
+english_name: CC Search Portal (Legacy)
```

Comment:
The parenthetical `Legacy` could be removed since this project is now active again.